charts/kube-plex/README.md

```diff
@@ -29,12 +35,18 @@ The following tables lists the configurable parameters of the Plex chart and the
 | `ingress.tls`                  | Ingress TLS configuration | `[]` |
 | `rbac.create`                  | Create RBAC roles? | `true` |
 | `nodeSelector`             | Node labels for pod assignment | `beta.kubernetes.io/arch: amd64` |
+| `configMap.plexPreferences.enabled`             | Enable init script(and load configMap) that will read all environment variables starting with PLEX_PREFERENCE_ | `false` |
+| `configMap.plexPreferences.name`             | Name of configMap | `41-plex-preferences` |
```

Comment:
Why would a user want to configure/override this, given that it's created/managed/controlled by the Helm chart anyway? From what I can tell, the user doesn't ever have a need to set/override the name of this ConfigMap (only needing to set the options within it)